index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="/quiz.css">
+</head>
+<body>
+	<article>
+		<header>
+			<h1>Fender Guitar</h1>
+		</header>
+		<section>
+			<img src="/htmlquizimg/fender.jpg">
+			<p class="just">Fender Strat with Caramel Sunburst finish</p>
+			<p class="colored">Not Available</p>
+		</section>
+		<section  class="bold second">
```

Comment:
**Pro tip:** Try to avoid naming classes to reflect the sequence of an element in the existing HTML.  If the sequence ever changes, it would lead to confusing code.  The class name should reflect either the type of content displayed in an element, or a specific kind of style being added (e.g. the `bold` class you use)